### ISSUE TITLE
App not quitting once all windows are closed

### ISSUE THREAD

**Issue by alexvandesande** [ethereum/mist#829]
When you close the splash screen window, it will not quit the app, instead it keeps both geth and the Wallet running in the background

**Comment by luclu** [comment]
The only thing I fould in [electron's release-notes](https://github.com/electron/electron/releases) between 1.0.1 and 1.2.2 that looks related is:
> Fix unexpected behavior when calling app.quit() while there are BrowserWindows with closable set to false. https://github.com/electron/electron/issues/5701 ("Properly notify observers when window prevents close")

```
 void NativeWindowViews::Close() {
+  if (!IsClosable()) {
+    WindowList::WindowCloseCancelled(this);
+    return;
+  }
+
   window_->Close();
 }
```

**Comment by luclu** [comment]
fixed by PR https://github.com/ethereum/mist/pull/849.

**Comment by bigcoinme** [comment]
This is very annoying!
Looking forward to the next release!

**Comment by luclu** [comment]
Fixed in `0.8.0`.
